.env.example

```diff
@@ -50,4 +50,8 @@ GEOCODE_API_KEY="geocode-api-key"
 SENTRY_ORG="sentry-org"
 SENTRY_PROJECT="sentry-project"
 SENTRY_DSN="sentry-dsn"
+REFRESH_APPROACH_VERSION="v1" #possible values v1, v2
```

Comment:
I want to test this in our staging server. Can you remove this part with the ENV variable and leave option 2 as default

Reply:
ack

Reply:
if we don't pass the value by default it will consider v2 approach

Reply:
Still that should be removed from the .env.example file. Also the code that checks which mode is active should be removed: https://github.com/Shelf-nu/shelf.nu/pull/1118/files#diff-65810b1364bf0c64d3666ec80137196e42d96762a19db504aaf2f88141e2d3fbR83